manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -506,11 +506,14 @@ properties:
       auctioneer:
         api_url: http://auctioneer.service.cf.internal:9016
       etcd:
-        machines: [(( grab properties.etcd.advertise_urls_dns_suffix ))]
-        ca_cert: (( grab meta.secrets.consul_ca_cert ))
-        client_cert: (( grab secrets.consul_agent_cert ))
-        client_key: (( grab secrets.consul_agent_key ))
+        machines: []
+        ca_cert: ""
```

Comment:
the thing with empty quotes is that it looks like there was something supposed to be (e.g. fetched from some file or generated) but something silently failed and now you have empty quote. If you put there 'not_used', then you make it clear that it is not used, as 'not_used' is also clearly not a valid cert, so deployment would fail if it was to be used...